core/src/main/java/org/geysermc/geyser/item/Items.java

```diff
@@ -482,7 +482,7 @@ public final class Items {
     public static final Item RED_TERRACOTTA = register(new BlockItem(builder(), Blocks.RED_TERRACOTTA));
     public static final Item BLACK_TERRACOTTA = register(new BlockItem(builder(), Blocks.BLACK_TERRACOTTA));
     public static final Item BARRIER = register(new BlockItem(builder().rarity(Rarity.EPIC), Blocks.BARRIER));
-    public static final Item LIGHT = register(new BlockItem(builder().rarity(Rarity.EPIC), Blocks.LIGHT));
+    public static final Item LIGHT = register(new LightItem(builder().rarity(Rarity.EPIC), Blocks.LIGHT));
```

Comment:
We will need to add this to the mappings generator. You don't need to make a PR; one of us can handle it.

Reply:
> We will need to add this to the mappings generator. You don't need to make a PR; one of us can handle it.

No need to make a PR, which part do you mean?
What changes do I need to make?

Reply:
Is it like adding code to https://github.com/GeyserMC/mappings-generator/blob/master/src/main/java/org/geysermc/generator/javaclass/GenerateItemsClass.java#L22 ?

Reply:
Yes, but you don't have to make the PR yourself unless you want to. One of our team members can.

Reply:
Okay, I've undo changes of Items.java.